data/human/human missions.txt

```diff
@@ -7455,3 +7455,141 @@ mission "Adelita 5"
+
+
+mission "Syndicate Business 1"
+	name "Rescue Kidnapped Man in <waypoints>"
+	description "A mysterious man has been kidnapped by a heavy pirate warship in <waypoints>. Destroy the ship to rescue him."
+	minor
+	to offer
+		random < 40
+		credits > 5000000
+		"combat rating" > 350
+	source
+		government "Syndicate"
+		attributes "core"
+	waypoint "Moktar"
+	destination "Sunracer"
+	on offer
+		conversation
+			`You are walking near the spaceport and suddenly a computer terminal starts blinking and beeping when you pass it.`
+			choice
+				`	(Check it out.)`
+				`	(Ignore it.)`
+					decline
+			`	When you approach, you are greeted by plain text on the terminal's screen:`
+			`	"SOS: Kidnapped by pirates, can't respond. <waypoints> standard coordinates: -1.18143560, -0.01575673, 0.00755328. Life in danger, need urgent rescue. Cannot communicate, all signals monitored. Please help!"`
+			`	You note that the message doesn't have any information about the sender, but there wouldn't be if what the message contains is true. There are no shortage of pirates in <waypoints>.`
+			choice
+				`	"To <waypoints>!"`
+				`	(Ignore it and walk away.)`
+					decline
+			`	You return to your ship and prepare to leave.`
+				accept
+
+	on enter "Moktar"
+		"reputation: Independent (Killable)" = -1000
+		conversation
+			`When you enter <waypoints>, your sensors detect a pirate Vanguard, but no sign of any escape pod. Your ship is immediately hailed, and the pirate opens communication.`
+			`	The voice of the apparent captain of the <npc> sends you a hail. "You were unwise to get involved in our business. Hand over that pompous salesman Howard, and I'll let you live."`
+			choice
+				`	"Uhhh... hand over who?"`
+				`	"I don't negotiate with pirates!"`
+					goto negotiate
+				`	"You can take him, but I don't know where he is."`
+			label who
+			`	"Bah, you spineless buffoon! Your lies won't save you or that useless salesman. I'll find him in your ship's wreckage!"`
+				goto power
+			label negotiate
+			`	"Aye, then I won't negotiate with you! Prepare to meet your end!"`
+			label power
+			`	The Vanguard's engines roar to life as you see over half a dozen fixed guns take aim at your ship, ready to strike.`
+	npc kill
+		government "Independent (Killable)"
+		personality heroic staying vindictive
+			confusion 20
+		system "Moktar"
+		ship "Vanguard" "Golden Jewel"
+		conversation
+			`The <npc> has been destroyed. You receive a hail from a small escape pod drifting nearby. "Thank you, Captain, you've saved my life. I'm Howard. I can promise you a reward if you pick me up from this tin can and drop me off at <destination>."`
+			choice
+				`	"Sounds good."`
+					decline
+				`	"What's the deal with you and that pirate?"`
+			`	"That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to go to <planet>.`
```

Comment:
"Time to go to" sounds like Howard is planning a business trip somewhere from the back of my ship. Maybe:
```suggestion
			`	"That pirate was a potential investor in the firm I work for. We had a disagreement over the terms and it obviously got quite out of hand. I'll need to find a new business partner now..." He trails off. Time to get back to <planet>.`
```